packages/plugins/type-designer/src/components/elements-type-container/columns.svelte

```diff
@@ -58,7 +60,9 @@
         {:else}
           <h2>{column.name} (hidden)</h2>
         {/if}
-        <Button on:click={() => toggleColumnVisibility(index)}>X</Button>
+        <IconButton on:click={() => toggleColumnVisibility(index)}>
+          <IconWrapper icon={column.visible ? "visibility" : "visibility_off"} fillColor="rgb(81, 159, 152)" />
```

Comment:
Here and in a few places in the css this colour palette is hardcoded, encapsulate it in a primary css colour if possible